src/components/LoginPanel.tsx

```diff
@@ -31,6 +37,16 @@ const MiddleDiv = styled.div`
         -webkit-box-shadow: 0 0 0 9999px ${(props) => props.theme.main.background[2]} inset !important;
         -webkit-text-fill-color: ${(props) => props.theme.main.foreground[2]} !important;
     }
+
+    @media (min-width: 300px) {
```

Comment:
I feel this would look better if both of these min-width's were ~75px bigger

Reply:
Thoguh as always I am open to other opinions

Reply:
Changed!